scss/mixins/vendor/bootstrap/_functions.scss

```diff
@@ -0,0 +1,13 @@
+// We need this before we update to the latest version
+// Modified from https://github.com/twbs/bootstrap/blob/v4-dev/scss/_functions.scss
+$theme-color-interval: 8% !default;
+@function theme-color-level($color, $level: 0) {
```

Comment:
It seems that this function is not used anywhere in our code. Why do we need it?